src/stretchy.js

```diff
@@ -259,52 +268,57 @@ const svgSpan = function(group: ParseNode, options: Options): span {
 };
 
 const encloseSpan = function(
-    inner: span,
+    inner: domTree.span,
     label: string,
     pad: number,
     options: Options,
-): span {
+): domTree.span {
     // Return an image span for \cancel, \bcancel, \xcancel, or \fbox
     let img;
     const totalHeight = inner.height + inner.depth + 2 * pad;
 
     if (/(fbox)|(color)/.test(label)) {
         img = buildCommon.makeSpan(["stretchy", label], [], options);
 
-        if (label === "fbox" && options.color) {
-            img.style.borderColor = options.getColor();
+        if (label === "fbox") {
+            const color = options.color && options.getColor();
+            if (color) {
+                img.style.borderColor = color;
+            }
         }
 
     } else {
         // \cancel, \bcancel, or \xcancel
         // Since \cancel's SVG is inline and it omits the viewBox attribute,
         // its stroke-width will not vary with span area.
 
-        let attributes = [["x1", "0"]];
+        let attributes: {[string]: string} = {"x1": "0"};
         const lines = [];
 
         if (label !== "cancel") {
```

Comment:
Honestly, I don't understand the structure of the original logic. However, I've replicated it herein the conversion to object.

Reply:
I think this is the "strikethrough" case.

Reply:
This probably could be simplified by moving not reusing the same `attributes` object for both diagonals.  Maybe
```
if (/^x?cancel$/.test(label)) {
    lines.push(new domTree.lineNode({
        x1: 0,
        y1: 0,
        x2: "100%",
        y2: "100%",
    });
}

if (/^[bx]cancel$/.test(label)) {
    lines.push(new domTree.lineNode({
        x1: 0,
        y1: "100%",
        x2: "100%",
        y2: 0,
    });
}
```

Reply:
Ok, I'll simplify it as a tiny PR separately after doing more paranoid checks about what pathways can actually end up in the `else` block that contains this code.